src/base/utils/string.cpp

```diff
@@ -119,23 +57,25 @@ Utils::String::NaturalCompare::NaturalCompare()
 #endif
 }
 
-bool Utils::String::NaturalCompare::operator()(const QString &l, const QString &r)
+bool Utils::String::NaturalCompare::operator()(const QString &left, const QString &right)
 {
+    // case-insensitive comparison
```

Comment:
and I recall I added a switch to change case sensitive / case insensitive in the past (unfortunately it didn't got accepted).